src/TrackGenerator.cpp

```diff
@@ -1825,66 +1929,71 @@ void TrackGenerator::splitSegments(FP_PRECISION max_optical_length) {
     log_printf(ERROR, "Unable to split segments since "
 	       "tracks have not yet been generated");
 
-  int num_cuts, min_num_cuts;
-  segment* curr_segment;
+#pragma omp parallel
```

Comment:
And this segment splitting should be both with and without CMFD to ensure the CMFD `segment` attributes are properly specified.